### PR TITLE
playbooks: add Ubuntu bionic playbook

### PR DESCRIPTION
**Pull request by scollazo**
This connects to archivematica/Issues#119.

### CODE REVIEW

File: playbooks/archivematica-bionic/vars-singlenode-1.6.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
```
Comment (mamedin) [inline playbooks/archivematica-bionic/vars-singlenode-1.6.yml:1]: I think this file is not needed. There are no support for bionic on AM 1.6

**Review by mamedin — CHANGES_REQUESTED** [review]
The bionic-support commit in SS stable/0.11.x is missing and thus we can finish the test.

**Review by sevein — CHANGES_REQUESTED** [review]
You may want to ignore this one too:

    playbooks/archivematica-bionic/ubuntu-bionic-18.04-cloudimg-console.log

**Comment by scollazo** [comment]
@sevein , removed `playbooks/archivematica-bionic/ubuntu-bionic-18.04-cloudimg-console.log` and vars-singlenode-1.6.yml

File: playbooks/archivematica-bionic/.gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/roles
+/.vagrant
+/src
+/ubuntu-xenial-16.04-cloudimg-console.log
```
Comment (sevein) [inline playbooks/archivematica-bionic/.gitignore:4]: Shouldn't this entry be different? `/ubuntu-bionic-18.04-cloudimg-console.log`?

File: playbooks/archivematica-bionic/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        $ ansible-playbook -i .vagrant/provisioners/ansible/inventory/vagrant_ansible_inventory singlenode.yml \
+           -u vagrant \
+           --private-key .vagrant/machines/am-local/virtualbox/private_key \
+           --extra-vars="archivematica_src_dir=/vagrant/src archivematica_src_environment_type=development" \
+           --tags="amsrc-pipeline-instcode" \
+           -v
+        ```
+
+6. The ansible playbook `singlenode.yml` specified in the Vagrantfile will provision using qa branches of archivematica. To provision using the stable 1.5.x/0.8.x branches, replace "vars-singlenode-qa.yml" with "vars-singlenode-1.5.yml" in `singlenode.yml`. You can also modify create a custom vars file and pass it instead (to modify role variables to deploy custom branches, etc.)  
+
```
Comment (sevein) [inline playbooks/archivematica-bionic/README.md:56]: Is ^ outdated?

File: playbooks/archivematica-bionic/Vagrantfile [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+
+  config.vm.box = ENV.fetch("VAGRANT_BOX", "ubuntu/bionic64")
+
+  {
+    "am-local" => {
+      "ip" => "192.168.168.196",
```
Comment (sevein) [inline playbooks/archivematica-bionic/Vagrantfile:13]: We've been allocating unique IPs for the different playbooks/****** environments, should the value here be different too?

File: playbooks/archivematica-bionic/README.md (hunk 3 above)
Comment (scollazo) [reply playbooks/archivematica-bionic/README.md:56]: Updated, but we'll have to update it again once the stable/0.12.x branch is created

File: playbooks/archivematica-bionic/Vagrantfile (hunk 4 above)
Comment (scollazo) [reply playbooks/archivematica-bionic/Vagrantfile:13]: Use 192.168.168.198

File: playbooks/archivematica-bionic/.gitignore (hunk 2 above)
Comment (scollazo) [reply playbooks/archivematica-bionic/.gitignore:4]: Fixed

**Review by sevein — APPROVED** [review]

**Comment by scollazo** [comment]
Merged as 85d10321aacb86bf24e2ff06ec465c75c99bae75

**Pull request closed without merging**